qna/src/app/interface/basiccli.py

```diff
@@ -1,4 +1,7 @@
+from ..parser import write_to_json
+from ..parser.parser import preprocess
```

Comment:
I think this is an unused import?

Reply:
preprocess will be unusued actually, I removed it and put it inside the models instead. Write_to_json is a method though as when the user does not accept the suggestions it writes it to json via this method